database/sql/V057__add_timestamp_index_to_order_events.sql

```diff
@@ -0,0 +1,2 @@
+-- Adds an index on the 'timestamp' column of 'order_events' table to facilitate efficient periodic cleanups.
+CREATE INDEX order_events_by_timestamp ON order_events (timestamp);
```

Comment:
Not sure we should have an index to speed up an operation we do at most once per day (current granularity of the interval setting).

Reply:
Makes sense, removed for now